### PR TITLE
add: uutils-coreutils

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

**Review by korewaChino — APPROVED** [review]

**Review by korewaChino — DISMISSED** [review]

**Review by nothingneko — DISMISSED** [review]
LGTM!!

File: anda/tools/uutils-coreutils/uutils-coreutils.spec [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Name:			uutils-coreutils
+Version:		0.0.22
+Release:		1%?dist
+Summary:		Cross-platform Rust rewrite of the GNU coreutils
+License:		MIT
+URL:			https://github.com/uutils/coreutils
+Source0:		%url/archive/refs/tags/%version.tar.gz
+Requires:		glibc gcc-devel
```
Comment (lleyton) [inline anda/tools/uutils-coreutils/uutils-coreutils.spec]: gcc-devel is not a real package, what are you actually trying to pull in?

**Review by lleyton — CHANGES_REQUESTED** [review]

**Review by lleyton — CHANGES_REQUESTED** [review]
good and bad news

good: it installs now
bad: I'm not sure why we have the uu- prefix on the binaries (and don't symlink)

**Review by korewaChino — DISMISSED** [review]

**Comment by korewaChino** [comment]
Many packages require coreutils itself. We need to somehow make the dep solver know that uutils can replace them all.

**Review by lleyton — DISMISSED** [review]

**Review by lleyton — APPROVED** [review]

**Pull request merged**